spec/migrations/20150921204114_add_vmware_ro_datastores_to_hosts_storages_spec.rb

```diff
@@ -2,11 +2,38 @@
 require_migration
 
 describe AddVmwareRoDatastoresToHostsStorages do
+  class AddVmwareRoDatastoresToHostsStorages::HostsStorage < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    self.table_name = "hosts_storages"
+  end
+
+  let(:host_storages_stub) { migration_stub(:HostStorage) }
   let(:hosts_storages_stub) { migration_stub(:HostsStorage) }
```

Comment:
This stub is for before the migration that changes the table name from hosts_storages to host_storages.  There is already a stub for HostStorage in the migration.